src/mappings/pool.ts

```diff
@@ -202,7 +202,7 @@ export function handleSetup(event: LOG_SETUP): void {
   pool.baseToken = token.id
   pool.baseTokenWeight = weiToDecimal(
     event.params.baseTokenWeight.toBigDecimal(),
-    token.decimals
+    18
```

Comment:
Is there some background on why this is needed? The datatoken is made according to a template (which can change) so it could feasibly have a different number of decimals in future

Reply:
template is hardcoded to 18, it was wrong to used `token.decimals` . Deploying another pool contract is not as easy , if we do that we will need more refactoring anyway.

Reply:
Ok cool, sounds fair enough